### ISSUE TITLE
Exclude multi-task log_vars from weight decay — unfreeze Kendall balancer

### ISSUE THREAD

**Issue by Polichinel** [views-platform/views-hydranet#59]
The Kendall et al. (2018) multi-task loss log_vars are frozen at zero because AdamW weight_decay=0.1 regularizes them back to initialization. The original paper used SGD with no weight decay. Diagnosis confirmed across 8 production sweep runs — ALL log_vars at 0.000 regardless of training length (80-200 lessons).

Fix: separate optimizer param group for log_vars with weight_decay=0.0. ~5-line change in make() in training_engine.py:

    optimizer = AdamW([
        {'params': model_params, 'weight_decay': 0.1},
        {'params': multitaskloss.log_vars, 'weight_decay': 0.0},
    ], lr=0.001)

If insufficient, give log_vars a 10x higher learning rate (separate param group).

Papers:
- Kendall, A. et al. (2018). Multi-Task Learning Using Uncertainty to Weigh Losses. CVPR 2018. incoming/multi/1705.07115v3.pdf
- Kunstner, F. et al. (2024). Heavy-Tailed Class Imbalance and Why Adam Outperforms SGD. papers/Kunstner2024_HeavyTailedClassImbalanceAdam.pdf

**Comment by Polichinel** [comment]
**SUPERSEDED** — replaced by the ZINB distributional-head epic #97 (master checklist #104); recorded in C-139 of the risk register. Rationale: mooted — a single ZINB likelihood has no reg/cls balancer to freeze
